Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -951,7 +951,7 @@ command scriptComment
    # OK-2009-03-03 : Bug 7537 - Reformat the handler after commenting, this may be flawed,
    # but the proper solution is rather time-consuming.
```

Comment:
@livecodeali after messing with this for most of the day I came to the conclusion that we would need to do a complete overhaul of formatting in order to support just formatting the selection. Accordingly I have updated the PR to support multiple-handler ranges for handler mode.